code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -46,16 +84,16 @@
 	heldname = new_heldname
 	user.log_message("set \"[heldname]\" as a name in a cyborg reclassification board at [loc_name(user)]", LOG_GAME)
 
-/obj/item/borg/upgrade/rename/action(mob/living/silicon/robot/R, user = usr)
+/obj/item/borg/upgrade/rename/action(mob/living/silicon/robot/borg, user = usr)
 	. = ..()
 	if(.)
```

Comment:
we can make this better by turning this into an early return

```suggestion
/obj/item/borg/upgrade/rename/action(mob/living/silicon/robot/borg, user = usr)
	. = ..()
	if(!.)
		return . // we already implicitly do this with any `return` call but let's be explicit here // don't include this comment
		
	// rest of code goes on this indentation
```

just helps it read cleaner